os-autoinst-obs-auto-submit

```diff
@@ -93,9 +93,9 @@ update_package() {
     sed -i -e 's/^  \* $/  * Update to latest openQA version/' ./*.changes
     $osc ci -m "Offline generation of ${version}" $ci_args
     rc=0
+    wait_for_package_build "$target"
```

Comment:
How is that supposed to work outside the loop below?

Reply:
ops. I forgot that. because I was testing it in another branch. will fix